MIDAS/src/systems.h

```diff
@@ -3,12 +3,14 @@
 #include "sensors.h"
 #include "rocket_state.h"
 #include "data_logging.h"
+#include "./hardware/telemetry.h"
```

Comment:
don't include hardware stuff unconditionally in the main folder. Make a telemetry.h file in the main folder that ifdefs to pick which folder to include telemetry from